newrelic/resource_newrelic_infra_alert_condition.go

```diff
@@ -30,7 +30,7 @@ func thresholdSchema() *schema.Resource {
 	return &schema.Resource{
 		Schema: map[string]*schema.Schema{
 			"value": {
-				Type:     schema.TypeInt,
+				Type:     schema.TypeFloat,
```

Comment:
Unfortunately this has a bit of a ripple effect that breaks the build. This is due to the typings in the Go client (`go-newrelic`) requiring `Value` to be of type `int` [here](https://github.com/paultyng/go-newrelic/blob/master/api/types.go#L416).  

Although CI passed, it would fail on the `master` branch 😢

Reply:
@sanderblue @jthurman42  Looks like the go-client has been updated https://github.com/newrelic/newrelic-client-go/pull/71. So this pr is no longer build breaking right?

Reply:
The Terraform provider has not been fully ported to newrelic/newrelic-client-go yet.  That work is likely to start next week.  (Note: Synthetics was ported, just not the rest)